R/slice_derivation.R

```diff
@@ -0,0 +1,171 @@
+#' Execute a Derivation with Different Arguments for Subsets of the Input Dataset
+#'
+#' The input dataset is split into subsets (slices) and for each slice the
+#' derivation is called separately. Some or all arguments of the derivation
+#' may vary depending on the slice.
+#'
+#' @param dataset Input dataset
+#'
+#' @param derivation Derivation
+#'
+#' @param args Arguments of the derivation
+#'
+#'   A `param()` object is expected.
+#'
+#' @param ... A `derivation_slice()` object is expected
+#'
+#'   Each slice defines a subset of the input dataset and some of the parameters
+#'   or the derivation. The derivation is called on the subset with the
+#'   parameters specified by the `args` parameter and the `args` field of the
+#'   `derivation_slice()` object. If a parameter is specified for both, the
+#'   value in `derivation_slice()` overwrites the one in `args`.
+#'
+#' @details
+#'
+#'   For each slice the derivation is called on the subset defined by the
+#'   `filter` field of the `derivation_slice()` object and with the parameters
+#'   specified by the `args` parameter and the `args` field of the
+#'   `derivation_slice()` object. If a parameter is specified for both, the
+#'   value in `derivation_slice()` overwrites the one in `args`.
+#'
+#'   - Observations which matches for more than one slice are considered for the
```

Comment:
line 31 I think could be clearer with`Observations that match with more than one slice are only considered for the first matching slice`

Reply:
Updated